doc-content/enterprise-only/release-notes/rn-771-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,72 @@
+* {CENTRAL} latency is correlated with the number of group memberships [https://issues.redhat.com/browse/RHPAM-2708[RHPAM-2708]]
+* In the test scenario designer, wrong test results are shown if the rules are not fired [https://issues.redhat.com/browse/RHDM-1330[RHDM-1330]]
+
+endif::[]
+
+ifdef::DM[]
+
+== {CENTRAL}
+* An exception is thrown when a test scenario containing a tag is renamed without saving [https://issues.redhat.com/browse/RHPAM-2674[RHPAM-2674]]
+* In guided rule editor, `java.time.format.DateTimeFormatter` is added multiple times to the rules when you use the use `LocalDate` type [https://issues.redhat.com/browse/RHDM-1318[RHDM-1318]]
+* In the *Data* tab of Guided Rule Templates, enumerated values are not displayed in the multiple select drop-down [https://issues.redhat.com/browse/RHDM-1219[RHDM-1219]]
+
+endif::[]
+
+ifdef::PAM[]
+
+== Process engine
+
+* When an intermediate timer from the process is triggered, the event sub-process with the timer is fired [https://issues.redhat.com/browse/RHPAM-2864[RHPAM-2864]]
+* An error boundary event is unable to handle an exception thrown by reusable sub-process node [https://issues.redhat.com/browse/RHPAM-2782[RHPAM-2782]]
+* The test that starts with high number of processes along with the high timers, the server throws an `OutOfMemoryError` error before all the process instances are being created [https://issues.redhat.com/browse/RHPAM-2912[RHPAM-2912]]
+
+endif::[]
+
+ifdef::DM[]
+
+== Decision engine
+
+* In the executable model, `*.class` files are missing in the dynamically created `.jar` files which are based on ruleflow files [https://issues.redhat.com/browse/RHDM-1250[RHDM-1250]]
+* An alpha node based rules which are evaluated for expired events are not executed [https://issues.redhat.com/browse/RHDM-1235[RHDM-1235]]
+* The fired rule goes into an infinite loop when the canonical model is enabled [https://issues.redhat.com/browse/RHPAM-2877[RHPAM-2877]]
+
+endif::[]
+
+== Installer
+
+* When you run the {PRODUCT} installer the installation path step references {JWS} 5.1 instead of {JWS} {JWS_VERSION} [https://issues.redhat.com/browse/RHPAM-2745[RHPAM-2745]]
+
+ifdef::PAM[]
+
+== Integration
+
+* Springboot support for `@Autowired` beans in the `NotificationListener` is missing [https://issues.redhat.com/browse/RHPAM-2705[RHPAM-2705]]
+
+endif::[]
+
+== {OPENSHIFT}
+
+* In {CENTRAL}, you can not login with the newly created `adminPassword` [https://issues.redhat.com/browse/RHPAM-2777[RHPAM-2777]]
+* {CENTRAL} does not accept the the newly created `adminUser` and `adminPassword`. It allows you to login with old credentials [https://issues.redhat.com/browse/RHPAM-2762[RHPAM-2762]]
+
+ifdef::DM[]
+== DMN designer
+
+* The node data type is lost after you drag and drop in the data type editor [https://issues.redhat.com/browse/RHDM-1269[RHDM-1269]]
```

Comment:
Suggested rewrite:
The node data type is lost when you drag and drop in the data type editor.